command/ssh/checkHost.go

```diff
@@ -61,7 +65,27 @@ func checkHostAction(ctx *cli.Context) error {
 		return err
 	}
 
-	resp, err := client.SSHCheckHost(ctx.Args().First())
+	id, err := ca.LoadDefaultIdentity()
```

Comment:
A user won't have an identity certificate unless the CA is configured with mTLS support, so perhaps we should check that first, `client.Version()` will return a struct with the `RequireClientAuthentication` field. We should only add the token if this is required.